kmath-complex/src/commonMain/kotlin/kscience/kmath/complex/Complex.kt

```diff
@@ -42,74 +45,70 @@ private val PI_DIV_2 = Complex(PI / 2, 0)
 /**
  * A field of [Complex].
  */
-@OptIn(UnstableKMathAPI::class)
 public object ComplexField : ExtendedField<Complex>, Norm<Complex, Complex>, RingWithNumbers<Complex> {
-    override val zero: Complex = 0.0.toComplex()
-    override val one: Complex = 1.0.toComplex()
+    public override val zero: Complex by lazy { 0.0.toComplex() }
```

Comment:
Lazy will be much more expensive here than an assignment. I do not think that there is a difference, but probably better to keep it simple.